### DISCUSSION TITLE
Great work Team

### DISCUSSION THREAD

**Discussion by bonesy512** [BearStudio/start-ui-web#172] [General]
Ive looked high and low for a boilerplate to start my #buildinpublic journey. Alot of what you built i dreamed in my head for site functionality. Now to figure out user auth and creating a functional website.

**Comment by yoannfleurydev** [comment]
Thanks a lot, that means a lot to the team behind this starter <3

**Comment by ivan-dalmet** [comment]
Thanks @bonesy512 For Auth you can use NextAuth if you want to go full stack :)
